css/style.css

```diff
@@ -21,51 +24,73 @@ main {
     background: none;
 }
 
-
 /* ################ Lesson component ################ */
 .lesson {
     background-color: #e9ecef;
 }
 
+/* ################ Footer ################ */
+footer {
+    font-size: 70%;
+    font-family: 'stolzl-light';
+}
 
-/* ################ (bad) darkmode from @etsde ################ */
-
-@font-face {
-    font-family: 'stolzl-bold';
-    src: url('/font/stolzl.bold.ttf') format('truetype'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+footer a, a:hover, a:focus, a:active {
+    text-decoration: none;
+    color: inherit;
 }
 
-@font-face {
-    font-family: 'stolzl-book';
-    src: url('/font/stolzl.book.ttf') format('truetype');
+/* ################ Cat Gif ################ */
+.rounded-box img {
+    width: 500px;
+    max-width: 100%;
+    border-radius: 10px;
 }
 
-@font-face {
-    font-family: 'stolzl-light';
-    src: url('/font/stolzl.light.ttf') format('truetype');
+.rounded-box {
+    display: flex;
+    justify-content: center;
 }
 
-/* TODO: WOFF files - uncomment and add when available
+
+/* ################ Import Font ################ */
```

Comment:
new fonts (also support for old browsers)

Reply:
Are we using all of them?